aleph-client/src/pallets/committee_management.rs

```diff
@@ -145,6 +161,14 @@ impl<C: ConnectionApi + AsConnection> CommitteeManagementApi for C {
             .at(&addrs)
             .map_err(|e| e.into())
     }
+
+    async fn get_lenient_threshold_percentage(&self, at: Option<BlockHash>) -> Option<Perquintill> {
+        let addrs = api::storage().committee_management().lenient_threshold();
+
+        self.get_storage_entry_maybe(&addrs, at)
+            .await
+            .map(|lt| Perquintill::decode(&mut &*lt.encode()).unwrap())
```

Comment:
This should ideally use `decode_all` for a little bit of safety.

Reply:
Is it OK to unwrap and panic? We could log an error and fallback to some default instead?

Reply:
if we will really panic here I don't know what default to really use? Plus I don't see how we will ever panic here but I may be proven wrong in the future

Reply:
I'd default to something where users don't take the hit - i.e. full leniency :P

Reply:
The panic would probably mean that someone is using aleph-client incompatible with the chain version. I'm treating this library as something we use internally for tests, so I don't mind panicking.

Reply:
Ah, I didn't notice it's in `aleph_client` 🙈 I thought it's in the node. Agree then.

Reply:
How would that help? In such cases, we would lie to them that they are getting full rewards while they could be actually losing them. Also, there is an accepted plan to divide aleph-client into chain agnostic and rest. This fn would go to the other part which would be for 'internal' use only such as tests etc. We have already reverted changes that made it possible to push it into crates.io so it is already only used in tests

Reply:
added decode_all :)